multiversx_sdk_cli/cli_contracts.py

```diff
@@ -524,6 +548,13 @@ def _send_or_simulate(tx: Transaction, contract_address: Address, args: Any):
 
 
 def verify(args: Any) -> None:
+    response = input(
```

Comment:
This is a blocking request that can break any integration into automation flows.
I would suggest at least to add a forcing argument to ignore the input request.

Curios whether products like Spawnable are importing this feature or implementing their own.

Reply:
Added an arg to skip the prompt. Added more info in PR description